core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -363,6 +355,8 @@ object ConsumerPerformance extends LazyLogging {
     val dateFormat = new SimpleDateFormat(options.valueOf(dateFormatOpt))
     val hideHeader = options.has(hideHeaderOpt)
     val recordFetchTimeoutMs = options.valueOf(recordFetchTimeoutOpt).longValue()
+    val bootstrapServers = options.valueOf(bootstrapServersOpt)
+    val zkConnectionStr = options.valueOf(zkConnectOpt)
```

Comment:
These seem to be unused?

Reply:
They are used in the added unit tests in [`ConsumerPerformanceTest.scala`](https://github.com/apache/kafka/pull/5097/files#diff-7e297244dfeb67e4a86f095470b6fa92).